src/Node/EnumCaseDeclaration.php

```diff
@@ -16,7 +16,7 @@ class EnumCaseDeclaration extends Node {
     /** @var Token */
     public $caseKeyword;
 
-    /** @var QualifiedName */
+    /** @var QualifiedName|Token */
```

Comment:
I suspect this and others like it are rather `MissingToken` (?) in which case parameter types need to be updated in Parser as they are getting `Token`

Reply:
I implemented that class - that was a typo. It should just be Token, the QualifiedName should be removed

Reply:
done